Assets/HoloToolkit/Input/Scripts/InputSources/GamepadInput.cs

```diff
@@ -0,0 +1,282 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using UnityEngine;
+using UnityEngine.VR.WSA.Input;
+
+namespace HoloToolkit.Unity.InputModule
+{
+    /// <summary>
+    /// GamepadInput class maps Xbox gamepad buttons to the GestureRecognizer.
+    /// Gamepad button A press and release maps to tap gesture.
+    /// Gamepad button A pressed maps to hold started, completed, canceled gesture.
+    /// Gamepad button A pressed plus left joystick rotate maps to navigation gesture.
+    /// </summary>
+    public class GamepadInput : BaseInputSource
+    {
+        [Tooltip("Game pad button to press for air tap.")]
+        public string GamePadButtonA = "Fire1";
```

Comment:
I very much disagree.  This wouldn't work "out-of-the-box" without modifying the input manager first anyway, unless devs are getting a copy of the input manager from the toolkit.